File: tests/js/anthropic/test_openllmetry.ts
/**
 * Conformance test: OpenLLMetry (Traceloop) JS Anthropic instrumentation.
 */

import { AnthropicInstrumentation } from "@traceloop/instrumentation-anthropic";
import { run } from "./common";

function instrument(anthropicModule: any) {
  const instrumentation = new AnthropicInstrumentation();
  instrumentation.manuallyInstrument(anthropicModule.default);
}

run("OpenLLMetry JS: Anthropic Conformance Test", instrument).catch((e) => { console.error(e); process.exit(1); });

File: tests/python/anthropic/test_openllmetry.py
"""Conformance test: OpenLLMetry (Traceloop) Anthropic instrumentation."""

from common import run


def instrument():
    from opentelemetry.instrumentation.anthropic import AnthropicInstrumentor
    AnthropicInstrumentor().instrument()


if __name__ == "__main__":
    run("OpenLLMetry: Anthropic Conformance Test", instrument)
